webpack.config.js

```diff
@@ -1,58 +1,64 @@
-var webpack = require('webpack');
-var PROD = process.env.NODE_ENV === 'production';
+/* global __dirname: true */
+const webpack = require('webpack');
+const PROD = process.env.NODE_ENV === 'production';
```

Comment:
Let me throw the changes up, now. I agree. I think the `const` was a slip of the mind as the linter was yelling.

I'm not sure, but in my experience `const` usually works and a few other things, but it's best not to mix syntax, as you've stated.